internal/collection/collection.go

```diff
@@ -21,3 +21,11 @@ func Keys[K comparable, V any](m map[K]V) []K {
 
 	return s
 }
+
+func FirstKeyFromMap[K comparable, V any](m map[K]V) K {
```

Comment:
@helderjs is this used anywhere? Besides, this is dangerous as the iteration order of keys inside of a map is not guaranteed. There is semantically not such a thing like "first key from map" in Golang.

Reply:
I see this is being used in tests, please move it there and rename it to `randomKeyFromMap` and not export it.

Reply:
@helderjs no, i think this method will be tempted to be used internally and I would like to prevent that.